pretrain/validation.py

```diff
@@ -103,7 +104,10 @@ def compute_losses(
     losses = []
     for batch in batches:
         try:
-            inputs = batch.to(device)
+            # Each batch is a tensor(1, max_sequence_length) from the falcon dataset loader.
+            # We truncate the batch to the appropriate sequence length for the model here.
+            batch_truncated = batch[:, :sequence_length]
```

Comment:
Confirmed this worked as expected in local jupyter notebook. Will add further tests for pretrain as a whole before merging.

Reply:
Note this will get replaced shortly in follow up CL to just use 2 separate tokenizers.